drivers/tee/optee/optee.c

```diff
@@ -27,6 +30,14 @@ LOG_MODULE_REGISTER(optee);
  */
 #define TEE_OPTEE_CAP_TZ  BIT(0)
 
+/*
+ * Bitmap of the ongoing notificatons, received from OP-TEE. Maximum number is
+ * CONFIG_OPTEE_MAX_NOTIF. This bitmap is needed to handle case when SEND command
+ * was received before WAIT command from OP-TEE. In this case WAIT will not create
+ * locks.
+ */
+SYS_BITARRAY_DEFINE_STATIC(notif_bitmap, CONFIG_OPTEE_MAX_NOTIF);
```

Comment:
wouldn't it be better to put it to  `optee_driver_data` if you will have a few driver instances?
according to the GP TEE documentation it is possible to have a few TEEs, e.g. two optee OSs running in one board, but on their schemes there is only one driver instance per all TEEs:
![Screenshot from 2023-06-29 13-19-26](https://github.com/oleksiimoisieiev/zephyr/assets/27217224/8e21a00e-cbb1-4310-8c84-a8068ec27ef9)



as least I think it would be nice if you could save modularity of drivers if we have a few driver instances in a future